requirements-test.txt

```diff
@@ -1,5 +1,5 @@
 -r requirements.txt
-coverage==6.0
+coverage==5.5
```

Comment:
Just out of curiosity, why the downgrade?

Reply:
Because new version does not seem to be compatible with `coveralls` (yet)